client/src/app/api/log-info/route.ts

```diff
@@ -0,0 +1,16 @@
+import { NextResponse } from 'next/server';
+export async function GET() {
+  const publicVarKeys = Object.keys(process.env).filter((key) => key.startsWith('NEXT_PUBLIC'));
+  const publicVars = publicVarKeys.reduce((acc: Record<string, string>, key) => {
+    acc[key] = process.env[key] ?? '';
+    return acc;
+  }, {} as Record<string, string>);
+
+  return NextResponse.json({
+    lastCommit: {
+      message: process.env.VERCEL_GIT_COMMIT_MESSAGE,
+      SHA: process.env.VERCEL_GIT_COMMIT_SHA,
```

Comment:
These variables are only available if deployed to Vercel. For example, if you build your app locally and then run it, you will see we don't have the commit number.

I think that we will need to find another solution to find out what is the last commit. We cannot use the GitHub Action for this because Cirad does not use it to deploy the application.

If we can't find any solution for the commit, then maybe we can just add this endpoint with the information about the environment variables. That would already be beneficial for us.